examples/chip-tool/BUILD.gn

```diff
@@ -24,7 +24,8 @@ executable("chip-tool") {
     "commands/common/Command.cpp",
     "commands/common/Commands.cpp",
     "commands/pairing/PairingCommand.cpp",
-    "commands/payload/ParseCommand.cpp",
+    "commands/payload/AdditionalDataParseCommand.cpp",
+    "commands/payload/QRCodeParseCommand.cpp",
```

Comment:
Current Parse Command dynamically detects the type of input (ManualSetupCode vs. QRCode) and parse it a SetupPayload object.  Renaming to QRCodeParseCommand might be misleading.  

How about, extending the parse command to pick the new payload type using the input type ?
`chip-tool payload parse qr_code` ?
`chip-tool payload parse manual_input_code` ?
`chip-tool payload parse  additional_data` ?

or using a  command line flag 
`chip-tool payload parse -additional #####` ?

Reply:
synced offline with Liju, we agreed on :
chip-tool payload parse-setup-payload
chip-tool payload parse-additional-data-payload